mkosi/config.py

```diff
@@ -795,6 +794,61 @@ def cache_manifest(self) -> dict[str, Any]:
         return manifest
 
 
+def parse_ini(path: Path, only_sections: Sequence[str] = ()) -> Iterator[tuple[str, str, str]]:
+    section: Optional[str] = None
+    setting: Optional[str] = None
+    value: Optional[str] = None
+
+    for l in textwrap.dedent(path.read_text()).splitlines():
+        for c in ("#", ";"):
```

Comment:
While checking the value for inline comments, I noticed
```python
configparser.ConfigParser(
            delimiters="=",
            comment_prefixes="#",
            inline_comment_prefixes="#",
            empty_lines_in_values=True,
            interpolation=None,
            strict=False,
            dict_type=ConfigParserMultipleValues,
        )
```
so for us `;` wasn't a comment prefix anymore. That being said, let's maybe add the above definition as a comment at the top of the function, so that it's clear what it's roughly supposed to mimic.